assets/package.json

```diff
@@ -9,17 +9,17 @@
   },
   "dependencies": {
     "phoenix": "file:../deps/phoenix",
-    "phoenix_html": "file:../deps/phoenix_html",
-    "throttle-debounce": "^2.0.0"
+    "phoenix_html": "file:../deps/phoenix_html"
   },
   "devDependencies": {
     "@fortawesome/fontawesome": "^1.1.7",
     "babel-cli": "^6.26.0",
     "babel-core": "^6.26.3",
     "babel-eslint": "^8.2.3",
-    "babel-loader": "^7.1.4",
+    "babel-loader": "^7.1.5",
+    "babel-plugin-lodash": "^3.3.4",
```

Comment:
We don't need `babel-plugin-lodash` if we're also using `lodash-es`, since what the Babel plugin does is rewrite
```javascript
import { map, filter, times, noop } from 'lodash';
```
into
```javascript
import map from 'lodash/map';
import filter from 'lodash/filter';
import times from 'lodash/times';
import noop from 'lodash/noop';
```